shared/cfg/guest-os/Windows.cfg

```diff
@@ -382,3 +382,5 @@
             install_path = "C:\Program Files (x86)\JAM Software\HeavyLoad"
         i386, i686:
             install_path = "C:\Program Files\JAM Software\HeavyLoad"
+   timedrift.with_pvclock.shared_ntp_date.ntp.with_stop.base_date:
+       rtc_base=2006-6-17
```

Comment:
It is better to move this modification near to "timedrift :".  Just a suggestion, thanks!